stopify/src/runtime/suspend.ts

```diff
@@ -43,7 +43,7 @@ export class RuntimeWithSuspend {
     // Clear the saved continuation, or invoking .resumeFromCaptured() twice
     // in a row will restart the computation.
     this.continuation = badResume;
-    return this.rts.resumeFromSuspension(cont);
+    return this.rts.resumeFromSuspension(cont, () => { });
```

Comment:
In addition to what Joe is saying about stepping programs interacting weirdly with this, since we don't store the `onDone` callback this resumption isn't ever going to call it. This is only currently used by our web demo debugger (I think), but we may want to think about how to make events work in this context too.